web/styles/inputs.css

```diff
@@ -0,0 +1,95 @@
+.input-active-styles {
+    color: var(--color-text-input-focus);
+    background-color: var(--color-background-input-focus);
+    border-color: var(--color-border-input-focus);
+}
+
+.input-element-wrapper {
+    display: grid;
+    grid-template:
+        [input-element-start] "icon-left-margin input-icon icon-content-gap content button-content-gap input-button button-right-margin" auto [input-element-end] / [input-element-start] var(
+            --input-icon-left-margin
+        )
+        var(--input-icon-width) var(--input-icon-content-gap) minmax(0, 1fr)
+        var(--input-button-content-gap) var(--input-button-width) var(
+            --input-button-right-margin
+        )
+        [input-element-end];
+    align-items: center;
+
+    .input-element {
+        grid-area: input-element;
+        box-sizing: border-box;
+        padding: 0.1875em 0.5em; /* 3px at 16px/1em and 8px at 16px/1em */
+        font-size: var(--base-font-size-px);
+        font-family: "Source Sans 3 VF", sans-serif;
+        line-height: 1.25;
+        text-overflow: ellipsis;
+        color: var(--color-text-input);
+        background: var(--color-background-input);
+        border: 1px solid var(--color-border-input);
+        border-radius: 4px;
+        outline: none;
+        transition: all 0.1s linear;
```

Comment:
Let's not use `all` here, even if it gets reset on the line below. You can omit the property portion of the `transition` property entirely: https://developer.mozilla.org/en-US/docs/Web/CSS/transition#syntax

Reply:
Removed the "property" portion (all) of the "transition" property.